components/automate-ui/src/app/pages/policy/details/policy-details.component.scss

```diff
@@ -18,41 +18,10 @@ chef-tab-selector {
   }
 }
 
-thead {
-  font-size: 14px;
-  letter-spacing: 0.2px;
-  color: $chef-primary-dark;
-}
-
-th {
-  text-align: left;
-  padding: 0px;
-}
-
-tbody > .detail-row {
-  padding-top: 0px;
-  padding-bottom: 15px;
-}
-
-.detail-row {
-  display: flex;
-  color: $chef-primary-dark;
-}
-
-.id-column {
-  width: 33%;
-  padding-top: 0px;
-  padding-left: 0px;
-}
-
-.type-column {
-  width: 66%;
-  padding-top: 0px;
-  padding-left: 0px;
-}
-
-.three-dot-entry {
-  justify-content: flex-end;
+chef-table-new {
```

Comment:
[nit] Nothing wrong with this, but could do away with the outer wrapper. Specifying just `.three-dot-entry` has enough specificity. (And minor benefit, that limits future edits on chef-table to only html files.)

(Same comment for each scss file.)